### DISCUSSION TITLE
Mysql

### DISCUSSION THREAD

**Discussion by mallen910** [inventree/InvenTree#3759] [Q&A]
I am following the install, and i keep getting stuck on the "sudo apt-get install mysql-server libmysqlclient-dev" comes back saying this 
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libmariadb-dev-compat libmariadb-dev

Package mysql-server is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

any suggestions I tried installing the suggested package but when I run the next command  sudo mysql -u root
saysvvsudo: mysql: command not found)
any help with this would be great

**Comment by SchrodingersGat** [comment]
@mallen910 the "bare metal" install instructions are written around a particular linux flavor and may not be suitable for your system.

What OS are you running?

**Comment by mallen910** [comment]
Debian GNU/Linux 11 (bullseye)

**Reply by matmair** [reply]
Debian 11 does not including mysql-server in the repos, here is  guide to install MariaDB instead: https://www.digitalocean.com/community/tutorials/how-to-install-mariadb-on-debian-11

**Reply by matmair** [reply]
@mallen910 did this answer your question or do you need further assistance?
